chore(docs): port MORE notions (#180)
sapphiredev/website: docs/Guide/application-commands/application-command-registry/advanced/setting-global-behavior-when-not-identical.mdx

> title: Configuring the global behavior when not identical

Reviewer: ```suggestion
title: Configuring the global behavior when commands are not identical
```

---

> Configuring the global behavior is easy, just import the [`ApplicationCommandRegistry`][application-command-registry] class from `@sapphire/framework` and set it to your desired option. For example, to override the behavior to overwrite by default, you may write:

Reviewer: Changing this as we'll default to overwrite soon

```suggestion
Configuring the global behavior is easy! You just need to import the [`ApplicationCommandRegistries`][application-command-registries] object from `@sapphire/framework` and call the `setDefaultBehaviorWhenNotIdentical` method with the new desired default behavior. For example, to override the behavior to only log differences by default, you may write:
```

Author: I thougth so, wasn't totally sure yet. Will note that 👍

```ts
import { RegisterBehavior } from "@sapphire/framework";

> ApplicationCommandRegistries.setDefaultBehaviorWhenNotIdentical(RegisterBehavior.Overwrite);

Reviewer: ```suggestion
ApplicationCommandRegistries.setDefaultBehaviorWhenNotIdentical(RegisterBehavior.LogToConsole);
```
